infra-aws/__main__.py

```diff
@@ -1,17 +1,26 @@
 """A pulumi program to deploy asterion-digital infra on aws."""
 
 # Import dependencies
+import asterion_organization
```

Comment:
Hey @shawngerrard - As outlined in [this comment](https://github.com/asterion-digital/asterion-as-code/issues/15#issuecomment-1104216969) I was expected to see organisations managed as a separate `org - asterion` stack directory, not within `infra - aws`.

Reply:
oops - updated!